.github/workflows/lint.yml

```diff
@@ -14,17 +14,19 @@ jobs:
   lintrunner:
     runs-on: linux.20_04.16x
     steps:
+      - name: Checkout PyTorch
+        uses: pytorch/pytorch/.github/actions/checkout-pytorch@master
+        with:
+          submodules: false
+          fetch-depth: 1
+
       - name: Setup Python
         uses: actions/setup-python@v2
         with:
           python-version: 3.8
           architecture: x64
 
-      - name: Checkout PyTorch
-        uses: pytorch/pytorch/.github/actions/checkout-pytorch@master
-        with:
-          submodules: false
-
+      # Lintrunner is not cached because it might change frequently
```

Comment:
it's probably okay to cache lintrunner. We haven't need to release a new version in a while https://github.com/suo/lintrunner/commits/main